pcsx2/gui/MainFrame.cpp

```diff
@@ -57,11 +61,18 @@ void MainEmuFrame::UpdateStatusBar()
 {
 	wxString temp(wxEmptyString);
 
-	if (g_Conf->EnableFastBoot)
-		temp += "Fast Boot - ";
+#ifndef DISABLE_RECORDING
+	if (g_InputRecording.IsActive() && g_InputRecording.GetInputRecordingData().FromSaveState())
+		temp += "Base Savestate - " + g_InputRecording.GetInputRecordingData().GetFilename() + "_SaveState.p2s";
+	else
```

Comment:
I'm not sure how I feel about modifying the system menu.  I mean it makes sense for what you're trying to accomplish.  But I can't help but feel like a better way to go would be to:
- gray out the boot option / disable it
- add a base savestate option to the `Input Record` menu.
To me it feels better UX-wise, but I could very well be wrong.